app/templates/teachers/teacher-student-view.jade

```diff
@@ -0,0 +1,220 @@
+              span(data-i18n="teacher.student_in_progress") 
+              
+              svg(width=15, height=15 style="margin: 0 5px 0 15px")
+                rect(width=15, height=15,fill="rgb(92, 180, 208)")
+              span(data-i18n="teacher.class_average") 
+          
+          .graphs
+            each versionedCourse in view.classroom.get('courses') || []
+              - var course = _.find(view.courses.models, function(c) {return c.id === versionedCourse._id;});
+              if !course
+                - continue;
+              - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
+              - if ((instance && instance.hasMember(view.user)))
+                svg(id='visualisation-'+versionedCourse._id, width="1142", height="600")
+
+        
+        //- - console.log (view.levelData[1].levelIndex)
+        
+        //- h4 Actions:
+          // Assign courses from dropdown
+          // Export CSV
+          // Apply License
+          // Edit password
+          // Remove Student
+          
+          
+        //- .student-chart
+        //-   div.student-levels-progress
+        //-     //- if view.allLevelSessionMap
+        //-       //- - console.log (view.allLevelSessionMap)
+        //-     each versionedCourse in view.classroom.get('courses') || []
+        //-         - var course = _.find(view.courses.models, function(c) {return c.id === versionedCourse._id;});
+        //-         if !course
+        //-           - continue;
+        //-         - var instance = view.courseInstances.findWhere({ courseID: course.id, classroomID: view.classroom.id })
+        //-         - if (instance && instance.hasMember(view.user))
+        //-           if course
+        //-             div.course-row.alternating-background
+        //-               strong
+        //-                 div.course-info= course.get('name')
+        //-               - var coursePlaytime = 0
+        //-               - var levels = view.classroom.getLevels({courseID: course.id}).models
+        //-               each level, index in levels
+        //-                 - var levelNumber = view.classroom.getLevelNumber(level.get('original'), index + 1)
+        //-                 //- - console.log (level)
+        //-                 - var levelProgress = view.levelProgressMap[level.get('original')]
+        //-                 if view.levelSessionMap && view.levelSessionMap[level.get('original')]
+        //-                   - var session = view.levelSessionMap[level.get('original')];
+        //-                   - coursePlaytime += session.get('playtime')
+        //-                   //- - var levelAverage = view.allLevelSessionMap[level.get('original')]
+        //-                   p= levelNumber + ". " + level.get('name') + ": " + session.get('playtime') + " (average: " + ")"
+        //-               p= coursePlaytime 
```

Comment:
Remove this if it's not being used, or put a comment at the top explaining what it's for/why it's broken if you're saving it for later.

Same with other commented out code chunks.